test/blackbox-tests/test-cases/cram/cram-tab.t

```diff
@@ -0,0 +1,37 @@
+Sometimes a user may misconfigure their editor and end up putting a tab
+character before a command in a cram test instead of two spaces.
+
+Dune would register this as a comment, leaving the user confused as to why
+their command is not being run. The current behaviour is for a leading tab to
+be treated and replaced as / by two spaces.
```

Comment:
Took me a few reads to understand this last bit. I think it would be clearer as:

```suggestion
be treated as, and replaced by, two spaces during promotion.
```